Clarify the actor when reponses omit parameters (#1807)
httpwg/http-extensions: draft-ietf-httpbis-priority.md

This document defines the urgency(`u`) and incremental(`i`) parameters. When
receiving an HTTP request that does not carry these priority parameters, a
server SHOULD act as if their default values were specified.

> An intermediary that forwards requests and response can combine signals from
> both; see {{merging}}. Note that omission of parameters in responses is handled
> differently from omission in requests.

Reviewer: ```suggestion
An intermediary can combine signals from request and response that it forwards.
Note that omission of parameters in responses is handled differently from
omission in requests; see {{merging}}.
```

This might be less confusing, as by definition, an intermediary forwards messages.

Reviewer 2: subject verb agreement; but this is an improvement
```suggestion
An intermediary can combine signals from requests and responses that it forwards.
Note that omission of parameters in responses is handled differently from
omission in requests; see {{merging}}.
```

Author: Agreed, thanks both!
